contrib/test.sh

```diff
@@ -1,6 +1,6 @@
 #!/bin/sh -ex
 
-FEATURES="bitcoinconsensus use-serde serde-decimal"
+FEATURES="secp256k1 consensus use-serde,secp256k1 serde-decimal,secp256k1"
```

Comment:
Since we have a default feature now maybe we should test with `--no-default-features` too? I guess we should have tested the "no features" case even before this PR, but now most developers won't run this case by default and we might accidentally brick the no-secp mode.

Reply:
I think it's best to test with `--no-default-features` and then loop over all the features to enable them one by one.

BTW, Ideally it would also test all combinations of features. (at least in building)

Reply:
We need one run without any features and then loop over the feature combination list with `--no-default-features` active.

It already tests all combinations since `serde-decimal` includes `use-serde`.